tests/cli.rs

```diff
@@ -71,6 +72,22 @@ fn accepts_content_path() -> TestResult {
     Ok(())
 }
 
+// --------------------------------------------------
+#[test]
+fn fails_invalid_content_path() -> TestResult {
+    let path = &generate_temp_path();
+    let content_path = "manifests";
+
+    // content_path is defined and succeeds
```

Comment:
nit: this comment is misleading? If we are asserting failure

Reply:
yup copy/paste error. updated.